datafusion/src/physical_plan/cross_join.rs

```diff
@@ -207,6 +208,79 @@ impl ExecutionPlan for CrossJoinExec {
             }
         }
     }
+
+    fn statistics(&self) -> Statistics {
+        stats_cartesian_product(
+            self.left.statistics(),
+            self.left.schema().fields().len(),
+            self.right.statistics(),
+            self.right.schema().fields().len(),
+        )
+    }
+}
+
+/// [left/right]_col_count are required in case the column statistics are None
+fn stats_cartesian_product(
```

Comment:
actually it isn't because we already had [this](https://github.com/apache/arrow-datafusion/blob/bb82ca100b233653811d14a4cc18cad5e5bd7536/datafusion/src/optimizer/hash_build_probe_order.rs#L75-L78) in the previous version of the `HashBuildProbeOrder` optimizer.